File: app/src/main/java/ru/zakoulov/gallery/imageController/tasks/TaskResponseImages.java
package ru.zakoulov.gallery.imageController.tasks;

import java.util.List;

import ru.zakoulov.gallery.imageController.Image;

/**
 * Created by Илья on 04.05.2018.
 */
public interface TaskResponseImages {
    void responseImagesDownload(List<Image> images);
}

File: app/src/main/java/ru/zakoulov/gallery/imageController/tasks/TaskResponseImages.kt
package ru.zakoulov.gallery.imageController.tasks

import ru.zakoulov.gallery.imageController.Image

/**
 * Created by Илья on 04.05.2018.
 */
interface TaskResponseImages {
    fun responseImagesDownload(images: List<Image>)
    fun responseImagesError()
}
